homeassistant/components/broadlink/light.py

```diff
@@ -2,21 +2,17 @@
 from abc import ABC, abstractmethod
 import logging
 
-from broadlink.exceptions import BroadlinkException
-
 from homeassistant.components.light import (
     ATTR_BRIGHTNESS,
     ATTR_COLOR_TEMP,
     ATTR_HS_COLOR,
     COLOR_MODE_COLOR_TEMP,
     COLOR_MODE_HS,
     COLOR_MODE_UNKNOWN,
-    SUPPORT_BRIGHTNESS,
-    SUPPORT_COLOR,
-    SUPPORT_COLOR_TEMP,
     LightEntity,
 )
 from homeassistant.core import callback
+from broadlink.exceptions import BroadlinkException
```

Comment:
This was correct, need to move to the top because it is an external module.

Reply:
ah ok, pylint told me it should be at the bottom, changing it back now.